lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -65,6 +65,10 @@
     - C(present) implies that the database should be created if necessary.
     - C(absent) implies that the database should be removed if present.
     - C(dump) requires a target definition to which the database will be backed up. (Added in Ansible 2.4)
+      Note that in some PostgreSQL versions of pg_dump, which is an embedded PostgreSQL utility and is used by the module,
```

Comment:
I think you mentioned in #40424 that this happens in PostgreSQL versions < 12, should we add the version here?

Reply:
I thought about this, it may be fixed in minor/fix versions of supported versions like in 9.4.20 or 11.4. Moreover 12 version is not released yet and I compiled the devel branch of it

Reply:
I think better to mention it in general without connection to a specific release (or we might the release it after 12 is released)

Reply:
👍 makes sense to me.